### PR TITLE
Fix indentation with spaces, not tabs :)

### PR DESCRIPTION
**Pull request by TheNain38**

### CODE REVIEW

File: assets/nh_files/modules/keyseed.py [hunk 1]
```diff
@@ -1526,239 +1526,239 @@
...
+    print '''echo -ne "\\x08\\x00\\x00\\x00\\x00\\x00\\x00\\x00" > /dev/hidg0''' #windows key
+    print '''echo -ne "\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00" > /dev/hidg0'''
+    print '''sleep 1'''
+    findinlist("\x63", locale) #c
+    print '''sleep 1'''
+    findinlist("\x6d", locale) #m
+    print '''sleep 1'''
+    findinlist("\x64", locale) #d
+    print '''sleep 1'''
+        print '''echo left-ctrl left-shift return | hid-keyboard /dev/hidg0 keyboard'''
```
Comment (TheNain38) [inline assets/nh_files/modules/keyseed.py]: Don't merge yet there is an error in indentation here... I think my regex to replace tabs with spaces has messed up here

**Comment by TheNain38** [comment]
You can merge now

**Comment by TheNain38** [comment]
@jmingov Could you check this PR, please?

**Comment by jmingov** [comment]
Binky will do it dont worry ^^.

**Comment by binkybear** [comment]
Tested - working.

**Pull request merged**